cpp/include/Ice/InputStream.h

```diff
@@ -113,60 +75,13 @@ namespace Ice
          */
         InputStream(const CommunicatorPtr& communicator, std::pair<const std::byte*, const std::byte*> bytes);
```

Comment:
I kept 4 public constructors: with or without encoding, and with 2 flavors of byte buffers.